index.js

```diff
@@ -17,8 +17,8 @@ module.exports = (opts, types, defaults) => {
 		try {
 			npmPath = require.resolve('npm', {paths: paths.slice(-1)});
 		} catch (error) {
-			// Error will be thrown if module cannot be found.
-			// We should ignore that error.
+			// If npm module cannot be found, add warning about adding global config to overwrite builtin config
+			warnings.push('Load npm buildin configs failed, you can use "pnpm config ls" to show buildin configs. And then use "pnpm config --global set <key> <value>" to migrate configs from builtin to global.');
```

Comment:
I think it should be built-in, not buildin

Reply:
Yeah, that's my fault to mispell it.